vrk-kpa/suomidigi
# SUOPA-643 New layout

To test:
- `make fresh`
- Clear caches and check that the new layout gets applied and looks good.
